cl/lib/search_index_utils.py

```diff
@@ -49,14 +52,40 @@ def get_parties_from_case_name(case_name: str) -> list[str]:
     :return: A list of parties. If no valid separator is found, returns an
     empty list.
     """
-
-    valid_case_name_separators = [
-        " v ",
-        " v. ",
-        " vs. ",
-        " vs ",
-    ]
-    for separator in valid_case_name_separators:
+    for separator in VALID_CASE_NAME_SEPARATORS:
         if separator in case_name:
             return case_name.split(separator, 1)
     return []
+
+
+def get_parties_from_bankruptcy_case_name(case_name: str) -> list[str]:
```

Comment:
I'd suggest one of:

```
get_parties_from_case_name_bankr  # Suffixed
bankr_get_parties_from_case_name  # Namespaced
get_parties_from_bankr_case_name  # Shorter
```

I think I like the first best, since it keeps the name consistent. I think any are improvements though?

Reply:
I like the first suggestion and will rename the helper function